.github/workflows/e2e.yml

```diff
@@ -0,0 +1,44 @@
+name: e2e
+on: [push]
+jobs:
+  cypress:
+    runs-on: ubuntu-latest
+    env:
+      POSTGRES_URL: "postgresql://digitalmarketplace:digitalmarketplace@localhost:5432/digitalmarketplace"
+      COOKIE_SECRET: "foobar"
+      MAILER_HOST: "foobar"
+      MAILER_PORT: "321"
```

Comment:
The app wouldn't start unless I had these filled out, but these placeholders seem to work

Reply:
Tech debt: let's run something like [Sendria](https://github.com/msztolcman/sendria) so these configured values are real within the testing environment.

Reply:
https://github.com/button-inc/digital_marketplace/issues/113